client/asset/dcr/simnet_test.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 const (
-	walletPassword = "123"
+	walletPassword = "abc"
```

Comment:
Another thing I has stashed for simnet_test.go (I think we were both looking into these) was:

```diff
--- a/client/asset/dcr/simnet_test.go
+++ b/client/asset/dcr/simnet_test.go
@@ -58,6 +58,7 @@ func mineAlpha() error {
 }
 
 func tBackend(t *testing.T, name string, blkFunc func(string, error)) (*ExchangeWallet, *dex.ConnectionMaster) {
+       t.Helper()
        user, err := user.Current()
        if err != nil {
                t.Fatalf("error getting current user: %v", err)
@@ -93,6 +94,7 @@ type testRig struct {
 }
 
 func newTestRig(t *testing.T, blkFunc func(string, error)) *testRig {
+       t.Helper()
        rig := &testRig{
                backends:          make(map[string]*ExchangeWallet),
                connectionMasters: make(map[string]*dex.ConnectionMaster, 3),
@@ -109,6 +111,7 @@ func (rig *testRig) beta() *ExchangeWallet {
        return rig.backends["beta"]
 }
 func (rig *testRig) close(t *testing.T) {
+       t.Helper()
        for name, cm := range rig.connectionMasters {
                closed := make(chan struct{})
                go func() {
@@ -247,6 +250,7 @@ func TestWallet(t *testing.T) {
 
        confCoin := receipts[0].Coin()
        checkConfs := func(n uint32) {
+               t.Helper()
                confs, err := rig.beta().Confirmations(confCoin.ID())
                if err != nil {
                        t.Fatalf("error getting %d confs: %v", n, err)
```

Wanna sneak those in too?